src/containers/line-mode.jsx

```diff
@@ -294,9 +291,6 @@ const mapStateToProps = state => ({
     isLineModeActive: state.scratchPaint.mode === Modes.LINE
 });
 const mapDispatchToProps = dispatch => ({
-    clearGradient: () => {
-        dispatch(clearStrokeGradient());
```

Comment:
Looks like this function might not be used anywhere anymore, and we could remove it?

Reply:
ok we'll just leave it for now since it's connected to the generic "color state" reducer generator